test/unit-tests/README.md

```diff
@@ -0,0 +1,2 @@
+Any UTs for vector-web layer components or functionality should go here.
+This used to contain the UTs for notifications before they got moved to react-sdk.
```

Comment:
Commit message is misleading here. Should there be changes to the tests as part of this commit?

Reply:
sorry, this was a terrible commit message. the thing that was breaking the tests is that the `unit-tests` dir had winked out of existence. creating this README file recreated the directory and fixed the tests O:-)